pkg/internal/gcp-get-valid-hostname.ps1

```diff
@@ -0,0 +1,25 @@
+# This script returns a mutated hostname if the length is longer than 63 characters. The hostname will be the
```

Comment:
`/gcp-get-hostname` will suffice. Valid is implied and obvious. If the script is returning an invalid hostname then it is a bug.

Reply:
+1, applies throughout PR